shoov/modules/custom/shoov_restful/plugins/restful/node/ci_builds/1.0/ShoovCiBuildsResource.class.php

```diff
@@ -91,10 +91,33 @@ public function patchEntity($entity_id) {
   protected function checkEntityAccess($op, $entity_type, $entity) {
     $account = $this->getAccount();
 
-    $wrapper = entity_metadata_wrapper('node', $entity);
-    $repo_name = $wrapper->og_repo->label();
+    if ($op == 'create' && $entity->is_new) {
+      // New CI Build is to be created.
+      $request = $this->getRequest();
+      if (!isset($request['repository'])) {
+        // Repository is not send.
```

Comment:
=> `Repository is not provided.` -- can this happen - isn't `repository` `required`?